keyboards/splitography/rules.mk

```diff
@@ -0,0 +1,40 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Processor frequency.
+#     This will define a symbol, F_CPU, in all source code files equal to the
+#     processor frequency in Hz. You can then use this symbol in your source code to
+#     calculate timings. Do NOT tack on a 'UL' at the end, this will be done
+#     automatically to create a 32-bit value in your source code.
+#
+#     This will be an integer division of F_USB below, as it is sourced by
+#     F_USB after it has run through any CPU prescalers. Note that this value
+#     does not *change* the processor frequency - it should merely be updated to
+#     reflect the processor speed set externally so that the code can use accurate
+#     software delays.
+F_CPU = 16000000
+
+#
+# LUFA specific
+#
+# Target architecture (see library "Board Types" documentation).
+ARCH = AVR8
+
+# Input clock frequency.
+#     This will define a symbol, F_USB, in all source code files equal to the
+#     input clock frequency (before any prescaling is performed) in Hz. This value may
+#     differ from F_CPU if prescaling is used on the latter, and is required as the
+#     raw input clock is fed directly to the PLL sections of the AVR for high speed
+#     clock generation for the USB and other AVR subsections. Do NOT tack on a 'UL'
+#     at the end, this will be done automatically to create a 32-bit value in your
+#     source code.
+#
+#     If no clock division is performed on the input clock inside the AVR (via the
+#     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
+F_USB = $(F_CPU)
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
```

Comment:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
STENO_ENABLE = yes
```

Reply:
Are most of these necessary? I'm getting the following warnings when compiling:

```
⚠ splitography: Feature bootmagic is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature mousekey is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature extrakey is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature console is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature command is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature nkro is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature backlight is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature audio is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature steno is specified in both info.json and rules.mk, the rules.mk value wins.
```

Reply:
If you have them in the info.json, then they're not needed here. 

But the additional stuff (such as F_CPU< ARCH, etc) are also not needed, and should be removed.

```suggestion
```